build/mvn

```diff
@@ -69,14 +69,23 @@ install_app() {
 
 # Install maven under the build/ folder
 install_mvn() {
-  local APACHE_MIRROR=${APACHE_MIRROR:-'https://www.apache.org/dyn/closer.lua?action=download&filename='}
+  MVN_VERSION="3.3.9"
```

Comment:
Thank you, @markhamstra .
I'll fix that to read "3.3.9" from pom file.

By the way, what do you mean by "make sure that the correct maven is downloaded and installed in the case that there isn't any mvn available when this script is run"?